tests/contrib/test_gce.py

```diff
@@ -66,11 +66,13 @@ def _refresh_success_helper(self, bytes_response=False):
         credentials.refresh(http)
         self.assertEquals(access_token, credentials.access_token)
 
-        base_metadata_uri = ('http://metadata.google.internal/0.1/meta-data/'
-                             'service-accounts/default/acquire')
+        base_metadata_uri = (
```

Comment:
My thought here is that it is usually better to repeat the URL here as that is the changed item (and if it changes in the future, we want to make sure the correct URL is being used).